app/bundles/FormBundle/Controller/FieldController.php

```diff
@@ -238,7 +238,10 @@ public function editAction($objectId)
                                     $aliases[] = $f['alias'];
                                 }
                             }
-                            $formField['alias'] = $this->getModel('form.field')->generateAlias($formField['label'], $aliases);
+                            $formField['alias'] = $this->getModel('form.field')->generateAlias(
+                                $formField['alias'] ? $formField['alias'] : $formField['label'],
```

Comment:
Hmm, my suggestion was not saved for some reason. Trying once more
```suggestion
                                $formField['alias'] ?? $formField['label'],
```